README.md

```diff
@@ -5,219 +5,722 @@ MailerSend Python SDK
 [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE.md)
 
 # Table of Contents
+
 * [Installation](#installation)
 * [Usage](#usage)
-* [Testing](#testing)
+  * [Sending and email](#sending_an_email)
+  * [Sending and email with CC and BCC](#cc_and_bcc)
```

Comment:
please update check all headings should match CURL header (H3) in doc

Reply:
readme updated according to Golang SDK